test/Auth0.OidcClient.Core.UnitTests/Tokens/IdTokenValidatorTests.cs

```diff
@@ -178,15 +178,6 @@ public async void ThrowsWhenIatMissing()
             Assert.Equal("Issued At (iat) claim must be an integer present in the ID token.", ex.Message);
         }
 
-        [Fact]
-        public async void ThrowsWhenIatIndicatesNotYetValid()
```

Comment:
What's your opinion around keeping this test, but now making it "not fail" if the claim has a bad value? This way you check the introduced change

Reply:
I don't think we should leave the future-tests in but negated - we're not making assertions or promises that all future tokens will always be honored, we're explicitly just not checking them.